mrp_bom_note/views/mrp_bom_view.xml

```diff
@@ -2,18 +2,18 @@
 <openerp>
     <data>
```

Comment:
@oihane I don't know if `noupdate` works in `<odoo>`, but you don't have any here so it should work.